src/css/header.css

```diff
@@ -1,3 +1,124 @@
 .header {
-  background-color: lightcoral;
+  padding-bottom: 8px;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  background-color: var(--white);
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.logo {
+  font-family: var(--font-family);
+  font-weight: 700;
+  font-size: 20px;
+  text-transform: lowercase;
+  color: var(--black);
+}
+
+.logo-web {
+  color: var(--green);
+}
+
+.nav,
+.menu-soc-list {
+  display: none;
+}
+
+@media screen and (min-width: 1280px) {
+  .nav,
+  .menu-soc-list {
+    display: flex;
+  }
+}
+
+.nav-list {
+  gap: 20px;
+}
+
+.nav-link {
+  position: relative;
+  font-weight: 600;
+  line-height: 1.28571;
+  color: var(--black);
+}
+
+.nav-link::after,
+.nav-link::after {
+  content: '';
+  display: block;
+  width: 37px;
+  height: 2px;
+  position: absolute;
+  bottom: -4px;
+  left: 0;
+  background-color: transparent;
+  transition: background-color var(--transition);
+}
+
+.nav-link:hover::after,
+.nav-link:focus::after {
+  background-color: var(--green);
+}
+
+.menu-soc-list {
+  gap: 12px;
+}
+
+.menu-soc-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 44px;

```

Comment:
Do not set width and height. Instead add `padding: 7px`

Reply:
Preffer to left as is